src/Components/ExpandControl/ExpandControlProps.ts

```diff
@@ -3,16 +3,20 @@ import * as PropTypes from "prop-types";
 export enum TriggerEvents {
     hover = "onMouseOver",
     click = "onClick",
+    out = "onMouseOut",
+    leave = "onMouseLeave",
```

Comment:
Does it events needed? Why changing expand state on mouseLeave?

Reply:
This is an abstract enumeration with trigger events. It is being used not only in ExpandControl. For example, popup should close on mouse leave. That's why there are the most common mouse events here

Reply:
Okay, if popup should not be controlled by itself (comment below), we need to put it as a child of ExpandControl. So we should change expand state to false on mouse leave event